MyString.java

```diff
@@ -20,8 +35,13 @@ public static void main(String args[]) {
      * @return the number of times c appears in str
      */
     public static int countChar(String str, char ch) {
-        //// Replace the following statement with your code
-        return 0;
+        int index = 0;
+        for(int i = 0; i < str.length(); i++){
+            if (str.charAt(i) == ch || str.charAt(i) == ch + 32 || str.charAt(i) == ch - 32 ) {
+                index++;
+            }

```

Comment:
32 is magic number 
use formula.. 
this is also not good.. consider the following: 
`countChar( "(Hello)", 'H')` -> will result in 2. 
since `(char)('H' - ('a' + 'A')) = '('`